src/Java/Images/Actor2D.md

```diff
@@ -0,0 +1,9 @@
+Description
+
+Example to demonstrate a actor that draws 2D data.
+
+The object vtkActor2D is similar to vtkActor, but it is made to be used with two dimensional images and annotation. 
+
+vtkActor2D object has a position but does not use a transformation matrix like vtkActor and it has a reference to a vtkMapper2D object which does the rendering.
+
+
```

Comment:
Looks to be an extra trailing blank line here, leave just one.

Reply:
is it ok now?